src/Servers/IIS/build/assets.props

```diff
@@ -11,217 +11,295 @@
-          PropetyName="AspNetCoreModuleV2OutOfProcessHandler"
-          OutDirName="OutOfProcessRequestHandler"
-          PackageSubPath="$(AspNetCoreModuleOutOfProcessVersion)\"
-          ProjectDirectory="$(MSBuildThisFileDirectory)..\AspNetCoreModuleV2\OutOfProcessRequestHandler"
-          Include="$(MSBuildThisFileDirectory)..\AspNetCoreModuleV2\OutOfProcessRequestHandler\OutOfProcessRequestHandler.vcxproj"
-          DllLocation="$(ArtifactsBinDir)OutOfProcessRequestHandler\$(NativeVCPlatform)\$(Configuration)\aspnetcorev2_outofprocess.dll"
+          VCPlatform="$(NativeVCPlatform)" 
+          Platform="$(NativePlatform)" 
+          ProjectDirectory="$(MSBuildThisFileDirectory)..\AspNetCoreModuleV2\OutOfProcessRequestHandler" 
+          ProjectName="OutOfProcessRequestHandler.vcxproj" 
+          PropetyName="AspNetCoreModuleV2OutOfProcessHandler" 
+          NativeAsset="aspnetcorev2_outofprocess" 
+          PackageSubPath="$(AspNetCoreModuleOutOfProcessVersion)\" 
+          BaseOutputPath="AspNetCoreModuleV2" 
+          OutDirName="OutOfProcessRequestHandler" 
+          DllLocation="$(ArtifactsBinDir)OutOfProcessRequestHandler\$(NativeVCPlatform)\$(Configuration)\aspnetcorev2_outofprocess.dll" 
+          Include="$(MSBuildThisFileDirectory)..\AspNetCoreModuleV2\OutOfProcessRequestHandler\OutOfProcessRequestHandler.vcxproj" 
           PdbLocation="$(ArtifactsBinDir)OutOfProcessRequestHandler\$(NativeVCPlatform)\$(Configuration)\aspnetcorev2_outofprocess.pdb"
         />
 
        <RunInProcessComponents
-          Platform="$(NativePlatform)"
-          VCPlatform="$(NativeVCPlatform)"
-          BaseOutputPath="AspNetCoreModuleV2"
-          NativeAsset="aspnetcorev2_inprocess"
-          ProjectName="InProcessRequestHandler.vcxproj"
-          PropetyName="AspNetCoreModuleV2InProcessHandler"
-          OutDirName="InProcessRequestHandler"
-          ProjectDirectory="$(MSBuildThisFileDirectory)..\AspNetCoreModuleV2\InProcessRequestHandler"
-          Include="$(MSBuildThisFileDirectory)..\AspNetCoreModuleV2\InProcessRequestHandler\InProcessRequestHandler.vcxproj"
-          DllLocation="$(ArtifactsBinDir)InProcessRequestHandler\$(NativeVCPlatform)\$(Configuration)\aspnetcorev2_inprocess.dll"
+          VCPlatform="$(NativeVCPlatform)" 
+          Platform="$(NativePlatform)" 
+          ProjectDirectory="$(MSBuildThisFileDirectory)..\AspNetCoreModuleV2\InProcessRequestHandler" 
+          ProjectName="InProcessRequestHandler.vcxproj" 
+          PropetyName="AspNetCoreModuleV2InProcessHandler" 
+          NativeAsset="aspnetcorev2_inprocess" 
+          BaseOutputPath="AspNetCoreModuleV2" 
+          OutDirName="InProcessRequestHandler" 
+          DllLocation="$(ArtifactsBinDir)InProcessRequestHandler\$(NativeVCPlatform)\$(Configuration)\aspnetcorev2_inprocess.dll" 
+          Include="$(MSBuildThisFileDirectory)..\AspNetCoreModuleV2\InProcessRequestHandler\InProcessRequestHandler.vcxproj" 
```

Comment:
How did you generate the changes to this file?

Reply:
src/Servers/IIS/tools/GenerateNativeAssets.ps1

Reply:
@wtgodbe do you think I should just hand generate this to make the diff more compact since it looks like the script changed the ordering which causes way more diffs

Reply:
No, this is fine.